### ISSUE TITLE
Shorten alerts for V and R sliders

### ISSUE THREAD

**Issue by terracoda** [phetsims/ohms-law#101]
Based on screen reader comments of alerts being verbose, after implementation of description changes in #90, I think it is safe to reduce the slider alerts by removing the repeated slider value information at the very end of the alert. 

Users get the slider value on focus and on a value change just before the actual alert. That seems to be sufficient.

I have updated the [Ohm's Law design document](https://docs.google.com/spreadsheets/d/15LuKXlzkh6xHpNbWC4ujtOfz1F7FJ7tj4oUF8Jvmj-k/edit#gid=0) by striking our the part of the alert that should be removed.

For clarity I repeat it here:
**Moving slider V slider up:**
- As letter V {{grows}}, letter I {{grows}}. Current now {{10.8 millamps}} with voltage at {{5.4 volts}}. 
  - Change to: As letter V {{grows}}, letter I {{grows}}. Current now {{10.8 millamps}}.

**Moving V slider down:**
-  As letter V {{shrinks}}, letter I {{shrinks}}. Current now {{7.2 millamps}} with voltage at {{3.6 volts}}. 
   - Change to: {{3.6 volts.}} As letter V {{shrinks}}, letter I {{shrinks}}. Current now {{7.2 millamps}}.

**Moving R slider up:**
- As letter R, {{grows}}, letter I, {{shrinks}}. Current now {{7.5 millamps}} with resistance at {{600 ohms}}.
  - Change to: As letter R, {{grows}}, letter I, {{shrinks}}. Current now {{7.5 millamps}}.

**Moving R slider down:**
- As letter R, {{shrinks}}, letter I, {{grows}}. Current now {{11.3 millamps}} with resistance at {{400 ohms}}.
  - Change to: As letter R, {{shrinks}}, letter I, {{grows}}. Current now {{11.3 millamps}}.

**Comment by terracoda** [comment]
@jessegreenberg, once this is ready, it would be great if @zepumph can update the sonified version.

**Comment by jessegreenberg** [comment]
@zepumph you have been taking the lead on a11y descriptions for this sim, do you have time to work on this?

**Comment by zepumph** [comment]
@terracoda could you let me know when the next interview will be that you would want the sonification wrapper updated for?

**Comment by terracoda** [comment]
@zepumph, I've got nothing scheduled, so you can mark this accordingly in your work priorities. 

FYI, I am not planning on doing any more work on descriptions for Ohm's Law. I think they are in good shape.

**Comment by emily-phet** [comment]
I just checked if this had been implemented in https://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.17/ohms-law-a11y-view.html. It appears this is good to go. Closing.
